nibabel/parrec.py

```diff
@@ -148,7 +148,7 @@
     'Technique': ('tech',),
     'Scan resolution  (x, y)': ('scan_resolution', int, (2,)),
     'Scan mode': ('scan_mode',),
-    'Repetition time [ms]': ('repetition_time', float),
+    'Repetition time [ms]': ('repetition_time', float, None),
```

Comment:
Would you mind adding a comment at the top of parrec above `_hdr_key_dict` noting that None means "length of vector to be determined on read"?